features/support/env.rb

```diff
@@ -3,11 +3,13 @@
 Before do
   # Force ids to be printed unquoted for consistency
   set_environment_variable('SHELL', '/usr/bin/bash')
+end
 
-  if RUBY_PLATFORM =~ /java/ || defined?(Rubinius)
-    @aruba_timeout_seconds = 120
+Aruba.configure do |config|
+  config.exit_timeout = if RUBY_PLATFORM =~ /java/ || defined?(Rubinius) || (defined?(RUBY_ENGINE) && RUBY_ENGINE == 'truffleruby')
```

Comment:
Given it's like "every Ruby implementation but CRuby", a shorter check might be `if !defined?(RUBY_ENGINE) || RUBY_ENGINE == 'ruby'; 10; else; 120; end`